package.json

```diff
@@ -34,5 +34,8 @@
     "prepare.py",
     "start.py",
     "venvtools.py"
-  ]
+  ],
+  "dependencies": {
+    "grouper": "^2.1.0"
```

Comment:
What is that?

Reply:
Oh, should've gotten rid of that. There was an error with the first release of modelerfour grouping that resulted in a failure bc the extension grouper wasn't installed, but garrett resolved it